lib/cli.js

```diff
@@ -182,16 +182,11 @@ exports.run = () => {
 	if (conf.watch) {
 		try {
 			const watcher = new Watcher(logger, api, files, arrify(conf.source));
-			watcher.observeStdin(process.stdin);
+			watcher.observeStdin(process.stdin).then(() => {
+				logger.exit(0);
+			}).catch(handleWatchError);
```

Comment:
I don't quite see how the new `observeStdin` would throw this error where the old one doesn't. Why does it need to be handled here?

Reply:
Yes. The code still needs some love. `observeStdin` is returning a promise that is not rejecting.
Will fix that.